modules/service/src/main/scala/lucuma/odb/service/ObservationService.scala

```diff
@@ -220,26 +305,33 @@ object ObservationService {
           SET.activeStatus.map(upActive),
           SET.constraintSet.toList.flatMap { cs =>
             List(
-              upCloud(cs.cloudExtinction),
-              upImage(cs.imageQuality),
-              upSky(cs.skyBackground),
-              upWater(cs.waterVapor)
-            )
+              cs.cloudExtinction.map(upCloud),
+              cs.imageQuality.map(upImage),
+              cs.skyBackground.map(upSky),
+              cs.waterVapor.map(upWater)
+            ).flattenOption
           }
         ).flatten
 
-      NonEmptyList.fromList(ups)
+      SET
+        .constraintSet
+        .flatMap(_.elevationRange)
+        .toList
+        .flatTraverse(elevationRangeUps)
+        .map { ers => NonEmptyList.fromList(ers ++ ups) }
     }
 
     def updateObservations(
       SET:   ObservationPropertiesInput,
       which: AppliedFragment
-    ): Option[AppliedFragment] =
-      updates(SET).map { us =>
-        void"UPDATE t_observation " |+|
-        void"SET " |+| us.intercalate(void", ") |+| void" " |+|
-        void"WHERE t_observation.c_observation_id IN (" |+| which |+| void")" |+|
-        void"RETURNING t_observation.c_observation_id"
+    ): Either[String, Option[AppliedFragment]] =
+      updates(SET).map {
+        _.map { us =>
+          void"UPDATE t_observation " |+|
```

Comment:
Is this skunk syntax?

Reply:
Yeah, `void` makes a Skunk `AppliedFragment` out of a bit of SQL that doesn't involve any parameters.  The `sql` interpolator is used when there are query arguments.  At least, that's my understanding.